bin/pg_repack.c

```diff
@@ -1948,6 +1961,10 @@ lock_exclusive(PGconn *conn, const char *relid, const char *lock_query, bool sta
 				pgut_rollback(conn);
 			else
 				pgut_command(conn, "ROLLBACK TO SAVEPOINT repack_sp1", 0, NULL);
+			if (timeout_msec < wait_msec)
+				usleep(1000 * (wait_msec - timeout_msec));
+			if (apply_log_table)
+				apply_log(conn, apply_log_table, 0);
```

Comment:
It is a good idea to apply log during attempts of acquiring lock to minimize duration of ACCESS EXCLUSIVE lock.

Although I'm trying to think of a way to avoid adding new argument `apply_log_table` in addition to `relid` and `lock_query`. 
Having just only these arguments would be nice:
```
static bool
lock_exclusive(PGconn *conn, const repack_table *table, bool start_xact, bool apply_log)
```

In most cases instance of `repack_table` already has relation OID and lock query. Except this case:
https://github.com/reorg/pg_repack/blob/5ae252fdd20e6866994fa30865b8c5b5e3d3b9b4/bin/pg_repack.c#L2181

Reply:
@ncleaton Any idea about the suggestions from @za-arthur?

Reply:
Hmm, to make that work repack_table_indexes would need to construct a repack_table struct, which seems confusing  when the table itself is not being repacked. I don't think it's justified to reduce the arg count by one.